sample-apps/discounts-tutorial/web/frontend/hooks/useUpdateDiscount.js

```diff
@@ -34,31 +35,36 @@ const UPDATE_CODE_MUTATION = gql`
   }
 `;
 
-export function useUpdateDiscount(discountMethod) {
-  const [triggerMutation, { isLoading }] = useShopifyMutation({
-    query: discountMethod === DiscountMethod.Automatic ? UPDATE_AUTOMATIC_MUTATION : UPDATE_CODE_MUTATION,
-  });
+export function useUpdateDiscount(method) {
+  const updateAutoDiscount = useShopifyMutation({
+    query: UPDATE_AUTOMATIC_MUTATION,
```

Comment:
Just curious: why did you decide to split the `useShopifyMutation` into two hook calls?